src/index.ts

```diff
@@ -11,6 +32,55 @@ class AngularJestTransformer extends TsJestTransformer implements Transformer {
   ): TransformedSource | string {
     return super.process(input, filePath, jestConfig, transformOptions);
   }
+
+  getCacheKey(
+    fileContent: string,
+    filePath: string,
+    _jestConfigStr: string,
+    transformOptions: CacheKeyOptions,
+  ): string {
+    this.createOrResolveTransformerCfg(transformOptions.config);
+
+    return createCacheKey()(fileContent, filePath, this._ngJestCfgStr, {
+      config: transformOptions.config,
+      instrument: false,
+    });
+  }
+
+  private createOrResolveTransformerCfg(jestConfig: Config.ProjectConfig): void {
```

Comment:
@wtho I added this new method and remove `configsFor`. The purpose is because `_ngJestCfgStr` is only scoped to this class so I don't want to put it into `NgJestConfig` like originally `ts-jest` does with its cache key.

Reply:
Looks good!